snap/quota/resources_test.go

```diff
@@ -46,8 +47,8 @@ func (s *resourcesTestSuite) TestQuotaValidationFails(c *C) {
 		{quota.NewResourcesBuilder().Build(), `quota group must have at least one resource limit set`},
 		{quota.NewResourcesBuilder().WithCPUCount(1).Build(), `invalid cpu quota with count of >0 and percentage of 0`},
 		{quota.NewResourcesBuilder().WithCPUCount(2).WithCPUPercentage(100).WithAllowedCPUs([]int{0}).Build(), `cpu usage 200% is larger than the maximum allowed for provided set \[0\] of 100%`},
-		{quota.NewResourcesBuilder().WithJournalRate(0, 1).Build(), `journal quota must have a rate count and period larger than zero`},
-		{quota.NewResourcesBuilder().WithJournalRate(1, 0).Build(), `journal quota must have a rate count and period larger than zero`},
+		{quota.NewResourcesBuilder().WithJournalRate(0, 1).Build(), `journal quota must have a rate count larger than zero and period at least 1 microsecond \(minimum resolution\)`},
+		{quota.NewResourcesBuilder().WithJournalRate(1, 0).Build(), `journal quota must have a rate count larger than zero and period at least 1 microsecond \(minimum resolution\)`},
```

Comment:
you should have  also a test with Period = time.Nanosecond or something like that?

Reply:
Good point! Added this